src/apps/tap/tap.lua

```diff
@@ -68,7 +67,9 @@ function Tap:pull ()
       if ethernet:is_bcast(p.data) then
          counter.add(self.shm.rxbcast)
       end
+      p = packet.allocate()
    end
+   self.pkt = p
```

Comment:
I think we can omit these two lines if we use `link.transmit(l, packet.clone(p))`?